FirebaseInAppMessagingSwift.podspec

```diff
@@ -0,0 +1,34 @@
+Pod::Spec.new do |s|
+  s.name                    = 'FirebaseInAppMessagingSwift'
+  s.version                 = '0.1.0'
```

Comment:
Match the beta naming and versioning of the other Firebase*Swift.podspecs

Reply:
Fixed in b58a326.